sql_generators/glean_usage/glean_app_ping_views.py

```diff
@@ -193,13 +193,13 @@ def _generate_select_expression(
                         if node.get("mode", None) == "REPEATED":
                             # unnest repeated record
                             select_expr.append(f"""
-                                ARRAY_AGG(
-                                    (   SELECT
+                                (
+                                    SELECT ARRAY_AGG(
```

Comment:
this works, but so does `ARRAY(SELECT STRUCT(...) ...)`, which i think might have been what you were going for before